ui/src/app/pages/profile/profile-page.spec.tsx

```diff
@@ -62,6 +62,7 @@ describe('ProfilePageComponent', () => {
       projectId: 'aaa',
       publicApiKeyForErrorReports: 'aaa',
       enableEraCommons: true,
+      enableComplianceTraining: true
```

Comment:
why did this change?

Reply:
There seems to be 4 cards that are typically shown in the profile, this one was missing in the test. I wanted to test the profile page with all cards showing and with the controlled tier card not showing. I could remove this and adjust the expected results.